content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,474 @@
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-amd64.tar.gz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-arm64.tar.gz
+{{< /tab >}}
+{{< /tabs >}}
+
+```shell
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service
+```
+
+<!--
+```shell
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-amd64.tar.gz
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service 
+```
+-->
+
+Configure:
+
+```shell
+mkdir -p /etc/containerd
+
+cat << EOF > /etc/containerd/config.toml
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".cni]
+
+    # binaries
+    bin_dir = "/opt/cni/bin"
+
+    # configure pod networking
+    conf_template = "/etc/cni/cni.template"
+EOF
+```
+
+Install:
+
+{{< tabs name="install_containerd" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+tar Cxzvf /usr/local/ containerd-1.7.22-linux-amd64.tar.gz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+tar Cxzvf /usr/local/ containerd-1.7.22-linux-arm64.tar.gz
+{{< /tab >}}
+{{< /tabs >}}
+
+```shell
+mv containerd.service /etc/systemd/system/
+systemctl enable --now containerd.service
+```
+
+<!--
+```shell
+tar Cxzvf /usr/local/ containerd-1.7.22-linux-amd64.tar.gz
+mv containerd.service /etc/systemd/system/
+
+systemctl enable --now containerd.service
+```
+-->
+
+Quick test:
+
+```shell
+systemctl status containerd.service
+```
+
+Detailed service check:
+
+```shell
+journalctl -u containerd
+```
+
+### Kubelet
+
+Download the [latest stable release](/releases/download/) of the Kubelet.
+
+{{< tabs name="download_kubelet" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubelet"
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+```
+-->
+
+Configure:
+
+{{< note >}}
+This tutorial uses the `10.241.1.0` network. Make sure you don't have any
+conflicts with an existing network. If necessary, use a network that is available to
+you.
+{{< /note >}}
+
+```shell
+mkdir -p /etc/kubernetes
+
+cat << EOF > /etc/kubernetes/kubelet.yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false
+authorization:
+  mode: AlwaysAllow
+enableServer: false
+logging:
+  format: text
+  sanitization: false
+readOnlyPort: 10250
+podCIDR: 10.241.1.0/24
+staticPodPath: /etc/kubernetes/manifests
+EOF
+```
+
+Install:
+
+```shell
+chmod +x kubelet
+mv kubelet /usr/bin/
```

Comment:
- likely to need `sudo`
- for tidiness, set the owner as `root`

Reply:
I've changed all instances of `sudo mv...` to `sudo cp...`, it saves the extra `chown` step, and you keep the sources intact, in case you want to reuse them